frontend/src/screens/Start.tsx

```diff
@@ -129,6 +94,11 @@ export default function Start() {
               <LoadingButton type="submit" loading={loading}>
```

Comment:
Should we then also change the title of the `TwoColumnLayoutHeader` to Start as well?